src/locale.en.js

```diff
@@ -12,11 +12,59 @@ export const login = {
 
 export const campsite = {
     nav_header: "Share a site, Get a site",
-    description: "Looking for some secret sites? Want to find a hidden spot? Upload a campsite, and if it's approved, we'll unlock two secret sites that are only available to our contributing members.",
+    site_description: "Looking for some secret sites? Want to find a hidden spot? Upload a campsite, and if it's approved, we'll unlock two secret sites that are only available to our contributing members.",
     upload: "Upload Campsite",
 
 };
 
 export const more_screen = {
   right_nav: 'Log Out'
-};
\ No newline at end of file
+};
+
+export const campsite_form = {
+  latitude: 'Latitude',
+  latitude_placeholder: 'add longitude',
+  longitude: 'Longitude',
+  longitude_placeholder: 'add longitude',
+  add_site_title:'Add Site Title',
+  add_site: 'Add a Site',
+  add_a_campsite: 'Add a Campsite',
+  site_info:'Site Info',
+  description: 'Description',
+  description_placeholder: 'Add Description',
+  directions: 'Directions',
+  directions_placeholder:'Add descriptive directions',
+  nearest_town:'nearest_town',
+  nearest_town_placeholder:'Enter nearest town',
+  here_now:'I am here now',
+  accessiblitiy: 'Accessiblitiy',
+};
+
+export const facilities_options = {
+  facilities:'facilities',
+  full_service: 'Full Service ',
+  some: 'Some',
+  none:'None',
+  permit: 'Permit',
+  paid: 'Paid',
+  free: 'Free'
+}
```

Comment:
Consider making these attributes in a nested object. So instead of 

```
 campsite_form = {...}
 // and
 facilities_options = {...}
```

Try doing something like: 

```
 campsite_form = {
  facilities_options: {...}
 }
```

This cleans up the export list, makes it clearer which texts are related, and makes your import on the corresponding components easier as well.

Reply:
And, actually, these could both be nested inside of `campsite`